ceno_zkvm/src/instructions/riscv/memory/store.rs

```diff
@@ -94,13 +93,13 @@ impl<E: ExtensionField, I: RIVInstruction, const N_ZEROS: usize> Instruction<E>
         let (new_memory_value, word_change) = match I::INST_KIND {
             InsnKind::SW => (rs2_read.memory_expr(), None),
             InsnKind::SH | InsnKind::SB => {
-                let change = MemWordChange::<N_ZEROS>::construct_circuit(
+                let change = MemWordUtil::<E, N_ZEROS>::construct_circuit(
```

Comment:
nitpick: rename `change` to reflect its real purpose?

Reply:
yes, I make every term as `new_memory` or `new_memory_value` to make everything consistent